### PR TITLE
Clown boots

### PR DESCRIPTION
**Pull request by shelltitan**
Icon need replacement. #317

### CODE REVIEW

**Comment by shelltitan** [comment]
@mechmind Could you help me please to fix this pr?

**Comment by mechmind** [comment]
First, create backup branch (git branch clown-backup), but do not switch to it. Then invoke `git rebase -i HEAD^^` . Git will open an editor with commits to be edited. Just remove line with fps limit commit and exit an editor. Commit will be removed from this branch. Check the changes and if everything is ok (it should be), force-push branch to github (`git push origin +clown`). If something go wrong you can abort rebase with `git rebase --abort`.

**Comment by shelltitan** [comment]
@mechmind Well that's better in a way now but still not the best.

**Comment by mechmind** [comment]
Well, something went wrong :)
There other way to fix it. Checkout master branch, pull fresh update, create new branch (you can delete current with `git branch -D clown` and create new clown branch) and pick the only one commit with `git cherry-pick ac512d359ff82c`

**Comment by shelltitan** [comment]
@mechmind Thanks for your help!

**Comment by mechmind** [comment]
You are wellcome.

File: sources/core/objects/Clothes.h [hunk 1]
```diff
@@ -73,6 +73,14 @@ class OrangeBoots: public Cloth
     OrangeBoots(quint32 id);
 };
 ADD_TO_TYPELIST(OrangeBoots);
+class ClownBoots: public Cloth
```
Comment (kremius) [inline sources/core/objects/Clothes.h]: Add 1 empty line after `ADD_TO_TYPELIST(OrangeBoots);` - it looks better that way.

File: sources/core/objects/Human.cpp [hunk 2]
```diff
@@ -96,6 +96,12 @@ bool Human::TryMove(Dir direct)
         {
             PlaySoundIfVisible("glass_step.ogg", GetOwner().Id());
         }
+        if (IdPtr<ClownBoots> shoes = GetHumanInterface()->feet_.Get())
+        {
+            unsigned int val = GetRand() % 2;
```
Comment (kremius) [inline sources/core/objects/Human.cpp]: This line can be removed.

Comment (shelltitan) [inline sources/core/objects/Human.cpp]: I didn't see I left this there.

**Pull request merged**